src/coreclr/src/gc/gcee.cpp

```diff
@@ -41,19 +41,19 @@ void GCHeap::UpdatePreGCCounters()
         hp = gc_heap::g_heaps [hn];
 
         allocation_0 +=
-            dd_desired_allocation (hp->dynamic_data_of (0))-
-            dd_new_allocation (hp->dynamic_data_of (0));
+            dd_desired_allocation (hp->dynamic_data_of (soh_gen0))-
+            dd_new_allocation (hp->dynamic_data_of (soh_gen0));
         allocation_3 +=
-            dd_desired_allocation (hp->dynamic_data_of (max_generation+1))-
-            dd_new_allocation (hp->dynamic_data_of (max_generation+1));
+            dd_desired_allocation (hp->dynamic_data_of (loh_generation))-
```

Comment:
seems odd that you have different code for WKS and SVR..

Reply:
odd. Also it seems that `allocation_0` and `allocation_3` are computed, but not used. .. 

---
In reply to: [368189884](https://github.com/dotnet/runtime/pull/1688#discussion_r368189884) [](ancestors = 368189884)

Reply:
I will just remove this code as unused.

---
In reply to: [368208845](https://github.com/dotnet/runtime/pull/1688#discussion_r368208845) [](ancestors = 368208845,368189884)

Reply:
Also this seems to be the last actionable comment on this PR that I am aware of.

---
In reply to: [368247232](https://github.com/dotnet/runtime/pull/1688#discussion_r368247232) [](ancestors = 368247232,368208845,368189884)

Reply:
ah, this was because the code for perf counters was removed and allocation_0/3 was used by perf counters but was mistakenly not removed.